understand/observed-size-spectra.qmd

```diff
@@ -94,14 +94,14 @@ The second way to deal with the fact that there are so many more small fish than
 ::: {.callout-note collapse="true"}
 ### Expand for R details
 
-We had decided that we wanted the breaks between the bins to increase with powers of 2. We first create the vector `log_breaks` with the exponents. The `seq()` function creates a vector of numbers starting at `from` and going up to `to` in steps of size `by`. You do not need to give the names of the arguments because they can also be identified by their position. So you could also have written `seq(0, 11, 1)`. Such integer sequences are used so often that there is the even shorter notation `0:11` giving the same sequence.
```

Comment:
I wasn't sure what you meant by "the breaks between the bins increase with powers of 2".